src/native/minipal/cpufeatures.c

```diff
@@ -551,6 +556,14 @@ int minipal_getcpufeatures(void)
         result |= ARM64IntrinsicConstants_Sve2;
     }
 
+    // TODO-PAC: Use PAC specific PF_* flag.
+    if (IsProcessorFeaturePresent(PF_ARM_V83_LRCPC_INSTRUCTIONS_AVAILABLE))
+    {
+        // Similar to FEAT_LRCPC, Pointer Authentication (FEAT_PAuth) is a mandetory feature in Armv8.3.
+        // Thus, we can imply availability of FEAT_PAuth when FEAT_LRCPC is present.
```

Comment:
I don't think it is safe to imply the presence of `FEAT_PAuth` should depend on the availability of a different extension.  There is `GetProcessMitigationPolicy(ProcessUserPointerAuthPolicy)` but from what I can tell, that tells the presence of user authentication at process level and is it safe to imagine that if this returns `true` `PAuth` is available on hardware. Or can someone turn the feature off at process level? @JasonLinMS - thoughts?